internal/pkg/aws/cloudwatchlogs/event.go

```diff
@@ -50,3 +51,17 @@ func (l *Event) shortLogStreamName() string {
 	}
 	return l.LogStreamName[0:shortLogStreamNameLength]
 }
+
+// colorCodeMessage returns the given message with color applied to every occurence of code
+func colorCodeMessage(message string, code string, givenColor *c.Color) string {
+	if c.NoColor {
+		return message
+	}
+	words := strings.Split(message, " ")
+	for i, word := range words {
+		if word == code {
+			words[i] = givenColor.Sprint(word)
```

Comment:
Btw, looks like this `ERR: ....` error message could not be shown with `ERR` to be in red right?

Reply:
I wonder if we can make use of regex to do this matching stuff https://play.golang.org/p/uPTVbBXzXFz

Reply:
Awesome thanks for all the feedback @iamhopaul123 and including a snippet of the regex!